system-test/system-test-script.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
calling this just `system-test` is good, no need for `script`

Reply:
fixed